chain/blockValidator.go

```diff
@@ -309,9 +309,9 @@ func SignerCheck(header *block.Header) error {
 		return fmt.Errorf("invalid block signer public key %x, should be %x", header.UnsignedHeader.SignerPk, publicKey)
 	}
 
-	_, err = crypto.DecodePoint(header.UnsignedHeader.SignerPk)
+	err = crypto.CheckPublicKey(header.UnsignedHeader.SignerPk)
```

Comment:
I think it will be more specific to rename it like `CheckPubLicKeySize/Length`

Reply:
Actually I was planning to put more pubkey check into that function if we have in the future, but I don't have anything in mind now.